python-sdk/src/configuration.rs

```diff
@@ -68,6 +72,23 @@ impl Configuration {
     fn get_flags_configuration(&self) -> Cow<[u8]> {
         Cow::Borrowed(self.configuration.flags.to_json())
     }
+
+    /// Return bytes representing bandits configuration.
+    ///
+    /// It should be treated as opaque and passed on to another Eppo client for initialization.
+    fn get_bandits_configuration(&self) -> PyResult<Option<Cow<[u8]>>> {
+        self.configuration
+            .bandits
```

Comment:
Possibly stupid question: is this the handle the downloaded (raw) bytes or are we re-serializing our internal representation?

(I'd prefer the former--just saving off and then returning when requested what we initialized with)

Reply:
It is re-serializing internal representation.

The flags configuration has been updated to hold the server response as is (it was actually done before our discussion took place) but bandits configuration has not been updated yet.

Reply:
Thanks for explaining that, makes sense.